backup-pruner.yml

```diff
@@ -46,3 +51,18 @@
     mode: "700"
   no_log: true
   with_items: "{{ TARSNAPPER_JOBS }}"
+
+- name: add parallel pruner script
+  template:
+    src: "backup-pruner.sh"
+    dest: "{{ BACKUP_PRUNER_SCRIPT }}"
+    owner: "root"
+    group: "root"
+    mode: "700"
+
+- name: add backup pruner cron
+  cron:
+    name: "backup pruner"
+    minute: 30
```

Comment:
Good call, I've adjusted this. Running all of the pruner scripts in parallel takes between 5 and 6 minutes consistently judging from roughly 10 trials. The backups themselves usually take a bit longer (but still < 15 minutes). I have nothing more than anecdotal evidence to back this up, though. :)